include/swift/Reflection/ReflectionContext.h

```diff
@@ -190,8 +193,112 @@ class ReflectionContext
 
     return true;
   }
-#endif // defined(__APPLE__) && defined(__MACH__)
-  
+#else // ELF platforms.
+  bool addImage(RemoteAddress ImageStart) {
+    auto Buf = this->getReader().readBytes(ImageStart, sizeof(Elf64_Ehdr));
```

Comment:
I think that we shouldn't assume 64-bitness.  32-bit ARM is pretty common still with Linux.  Byte 5 should identify the correct bitsex: 1 -> 32-bit, 2 -> 64-bit.  This is technically named `EI_CLASS`.

Reply:
My understanding is that there's no 32-bit support for swift (Linux). 
As such, I think we can generalize this as a follow up. Probably we should have an assertion `"unimplemented"` for anything that's 32-bits?